microsoft-edge/devtools-guide-chromium/memory-problems/index.md

```diff
@@ -251,6 +251,33 @@ DevTools shows you a breakdown of memory allocation by function.  The default vi
 ![Allocation sampling.](../media/memory-problems-glitch-example-05-memory-allocation-sampling-heavy-bottom-up.msft.png)
 
 
+<!-- ====================================================================== -->
+## Reduce garbage with additional settings for Allocation sampling
+
+By default, the **Allocation sampling** profiling type only reports allocations that are still alive at the end of the recording session.  Objects that are created, removed, and then garbage collected (GC'd) aren't displayed in the **Memory** tool when profiling using the **Allocation sampling** or **Allocation instrumentation on timeline** types.
+
+You can trust the browser to clean up garbage from your code.  However, it is important to consider that GC itself is an expensive operation and multiple GCs can slow down your user's experience of your website or app.  When recording in the **Performance** tool with the **Memory** checkbox turned on, you can see the GC operation happen at the steep cliffs in the heap chart.  
+
+![GC operation shown in the Performance tool.](../media/memory-problems-gc-in-performance.png)
+
+To reduce the amount of garbage your code is creating and reduce the cost or the number of times GC is run, you can configure the **Allocation sampling** profiling type to track objects that are discarded by GC with settings.
+
+1. Click the **Allocation sampling** radio button.
```

Comment:
```suggestion
1. Click the **Allocation sampling** option button.
```
check UI and check 
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/r/radio-button